sdk/wallet-adapter/packages/wallet-standard/README.md

```diff
@@ -0,0 +1,114 @@
+    // Return the version of the Wallet Standard this implements (in this case, 1.0.0).
+    return "1.0.0";
+  }
+  get name() {
+    return "Wallet Name";
+  }
+  get icon() {
+    return "some-icon-data-url";
+  }
+
+  // Return the Sui chains that your wallet supports.
+  get chains() {
+    return [SUI_DEVNET_CHAIN];
+  }
+}
+```
+
+### Implementing features
+
+Features are standard methods that can be used by consumers to interact with the wallet. For Sui wallets, we expect the following features to be implemented:
+
+- `standard:connect` - Used to initiate a connection to the wallet.
+- `sui:signAndExecuteTransaction` - Used to prompt the user to sign a transaction, then submit it for execution to the blockchain.
+
+These features can be implemented in your wallet class under the `features` property:
+
+```typescript
+import {
+  ConnectFeature,
+  SuiSignAndExecuteTransactionFeature,
+} from "@mysten/wallet-standard";
+
+class YourWallet implements Wallet {
+  get features(): ConnectFeature & SuiSignAndExecuteTransactionFeature {
+    return {
+      "standard:connect": {
+        version: "1.0.0",
+        connect: this.#connect,
+      },
+      "sui:signAndExecuteTransaction": {
+        version: "1.0.0",
+        signAndExecuteTransaction: this.#signAndExecuteTransaction,
+      },
+    };
+  },
+
+	#connect() {
+		// Your wallet's connect implementation
+	}
+
+	#signAndExecuteTransaction() {
+		// Your wallet's signAndExecuteTransaction implementation
+	}
+}
+```
+
+### Exposing accounts
+
+The last requirement of the wallet interface is to expose an `acccounts` interface. This should expose all of the accounts that a dapp has access to once it has connected. It can be empty prior to initiating a connection through the `standard:connect` feature.
+
+The accounts can use the `ReadonlyWalletAccount` class to easily construct an account matching the required interface.
+
+```typescript
+import { ReadonlyWalletAccount } from "@mysten/wallet-standard";
+
+class YourWallet implements Wallet {
+  get accounts() {
+    // Assuming we already have some internal representation of accounts:
+    return someWalletAccounts.map(
+      (walletAccount) =>
+        // Return
+        new ReadonlyWalletAccount({
+          address: walletAccount.suiAddress,
+          publicKey: walletAccount.pubkey,
+          // The Sui chains that your wallet supports.
+          chains: [SUI_DEVNET_CHAIN],
+          // The features that this account supports. This can be a subset of the wallet's supported features.
+          // These features must exist on the wallet as well.
+          features: ["sui:signAndExecuteTransaction", "standard:signMessage"],
+        })
+    );
+  }
+}
+```
+
+### Registering in the window
+
+Once you have a compatible interface for your wallet, you can register it in the window under the `window.navigator.wallets` interface. This is an array-like interface where all wallets self-register by pushing their wallet into.
```

Comment:
their wallet into is correct here? 
perhaps "Wallets self register by pushing their wallet to this array-like interface.

Reply:
I changed the terminology here, let me know if it makes sense